<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nosto</class>
 <widget class="QDialog" name="nosto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_nosto_omistaja">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_nosto">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Saldo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_nosto_saldo">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>Näyttää saldon</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_nosta20">
         <property name="text">
          <string>20 €</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nosta40">
         <property name="text">
          <string>40 €</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nosta60">
         <property name="text">
          <string>60 €</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nosta100">
         <property name="text">
          <string>100 €</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nosta200">
         <property name="text">
          <string>200 €</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nosta500">
         <property name="text">
          <string>500 €</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_sulje_nosto">
       <property name="text">
        <string>sulje</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_sulje_nosto</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_sulje_nosto</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
